Source/WebCore/platform/graphics/Color.cpp

```diff
@@ -183,8 +183,7 @@ bool Color::anyComponentIsNone() const
         if constexpr (std::is_same_v<ColorType, SRGBA<uint8_t>>) {
             return false;
         } else {
-            auto [c1, c2, c3, alpha] = underlyingColor.unresolved();
-            return std::isnan(c1) || std::isnan(c2) || std::isnan(c3) || std::isnan(alpha);
+            return underlyingColor.unresolved().anyComponentIsNone();
```

Comment:
Maybe take the braces out to match normal WebKit coding style.

Reply:
In an `if constexpr` I think the `else` has meaning even after a `return`. So I suggest keeping it.